ai-safety-debate/train_classifier_via_debate.py

```diff
@@ -54,6 +56,18 @@ def run(
     else:
         raise Exception("Unknown dataset in " + "dataset.txt: " + dataset)
 
+    judge_accuracy = judge.evaluate_accuracy()
+    print("Judge accuracy:", judge_accuracy)
+
+    if precomputed_debate_results_path is not None:
+        debate_results = np.fromfile(precomputed_debate_results_path).reshape(
+            -1, 10, 10
+        )
+        print("Loaded debate results from", precomputed_debate_results_path)
+        print("These will be used for training instead of re-running the debates.")
```

Comment:
Could be nice to assert that cheat_debate isn't true here

Reply:
Agreed, will do!